errors/errors.rb

```diff
@@ -1,7 +1,7 @@
-# frozen_string_literal: true
 
-module Errors # :nodoc:
-  class WrongClassError < StandardError # :nodoc:
+
+module Errors
```

Comment:
Layout/LeadingBlankLines: Unnecessary blank line at the beginning of the source.